src/main/resources/templates/animals/list-animals.html

```diff
@@ -1,64 +1,66 @@
-<!DOCTYPE html>
-<html lang="en" xmlns:th="http://thymeleaf.org">
-<head>
-<!-- Required meta tags -->
-<meta charset="utf-8">
-<meta name="viewport"
-	content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
-<!-- Bootstrap CSS -->
-<link rel="stylesheet"
-	href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
-	integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
-	crossorigin="anonymous">
-	
-<title>Animal Directory</title>
-</head>
-
-
-
-<body>
-<div class="container">
-
-
-	<h4>Pet Information Dictionary</h4>
-	
-	
-	
-	
-	
-	<hr>
-	
-	<!-- Create a button to add new Faculty -->
-	<a th:href="@{/Animals/viewAddForm}" class="btn btn-dark">Add New Animal</a>
-	
-	<table class="table table-bordered table-dark">
-		<thead>
-			<tr class="bg-danger">
-				<th>Name</th>
-				<th>Type</th>
-				<th>Update Info</th>
-			</tr>
-		</thead>
-
-		<tbody>
-			<tr th:each="theAnimal : ${animals}">
-				<td th:text="${theAnimal.name}" />
-				<td th:text="${theAnimal.type}" />
-				
-				<!-- Update Button -->
-				<td>
-				<a th:href="@{/Animals/viewUpdateForm(animalId=${theAnimal.id})}"
-				class="btn btn-light btn-sm">Update</a>
-				<!-- -Delete Button -->
-				<a th:href="@{/Animal/delete(animalId=${theAnimal.id})}"
-				class="btn btn-danger btn-sm"onclick="if(!(confirm('Please confirm that you to remove this animal')))return false">Delete</a>
-				</td>
-			</tr>
-		</tbody>
-
-	</table>
-
-</div>
-</body>
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://thymeleaf.org">
+<head>
+<!-- Required meta tags -->
+<meta charset="utf-8">
+<meta name="viewport"
+	content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+<!-- Bootstrap CSS -->
+<link rel="stylesheet"
+	href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
+	integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+	crossorigin="anonymous">
+	
+<title>Animal Directory</title>
+</head>
+
+
+
+<body>
+<div class="container">
+
+
+	<h4>Pet Information Dictionary</h4>
+	
+	
+	
+	
+	
+	<hr>
+	
+	<!-- Create a button to add new Faculty -->
+	<a th:href="@{/Animals/viewAddForm}" class="btn btn-dark">Add New Animal</a>
+	
+	<table class="table table-bordered table-dark">
+		<thead>
+			<tr class="bg-danger">
+				<th>Name</th>
+				<th>Type</th>
+				<th>Update Info</th>
+			</tr>
+		</thead>
+
+		<tbody>
+			<tr th:each="theAnimal : ${animals}">
+				<td th:text="${theAnimal.name}" />
+				<td th:text="${theAnimal.type}" />
+				
+				<!-- Update Button -->
+				<td>
+				<a th:href="@{/Animals/viewUpdateForm(animalId=${theAnimal.id})}"
+				class="btn btn-light btn-sm">Update</a>
+				<!-- -Delete Button -->
+				<!-- should have delete only if signed in -->>

```

Comment:
I added only a comment Because dont know what you guys want, I do make a suggestion that you should make it so that you can delete from the user(who's sign in) account so it doesn't delete everything.